src/main/java/aquarium/shop/UserDetails.java

```diff
@@ -0,0 +1,76 @@
+package aquarium.shop;
+
+import org.hibernate.annotations.ColumnTransformer;
+
+import javax.persistence.*;
+
+import static javax.persistence.GenerationType.IDENTITY;
+
+@Entity
+@Table(name = "user_details")
+public class UserDetails {
+
+    @Id
+    @GeneratedValue(strategy = IDENTITY)
+    private Integer id;
+
+    @Column(nullable = false)
+    private String name;
+
+    @ColumnTransformer(
```

Comment:
Maybe it's not the best solution as it will work only for Postgres. If you change DB provider to different one, it will require modification of this code. But let's leave it like that, we don't plan to change provider.